[W11-4] Book Squirrel

0blivious opened the pull request.

kylase-learning commented:
## Feedback (Week 7; v1.1)
 
### Github Project Page
- [x] Update Project description
- [x] Update Link to Project Page
 
### README
- [x] UI Mockup updated in project website
- [ ] Make the project website look like a product home page (aka remove the top nav bar)
- [x] AB4 acknowledgement
- [x] Travis badge updated to own fork's
- [x] Coveralls badge updated to own fork's (_the link has extra https:// as prefix_)
 
### [Project Page](https://cs2103-ay1819s2-w11-4.github.io/main/)
- [x] Update to remove AB4 reference
- [x] Travis badge updated to own fork's
- [x] Coveralls badge updated to own fork's (_the link has extra https:// as prefix_)
 
### About Us Page
- [x] Updated with photos
- [x] Roles and components assigned (_Could describe your individual roles more precisely_)

### Additional Stuffs
- Try to give descriptive messages for your commits instead of leaving in empty or "no message". If you think it is very time consuming and not productive, then you should batch your changes and commit as a collective set of cohesive changes. 
 
UG/DG feedback will be provided to you in the documents.

kylase-learning commented:
Please do fix your git setup as you can see from the commit history, it can't identify correct the author of the commit as one of you.

This will affect the ability of RepoSense to detect your contribution.

0blivious committed:
Merge pull request #89 from Miaaaaa97/pr/12

Pr/12

Commit:
update to show review of a certain book when click

Commit:
style fix

0blivious committed:
Merge pull request #90 from 0blivious/master

update to show review of a certain book when click

EleenYang committed:
update sample data

EleenYang committed:
Merge branch 'master' into master

0blivious committed:
Merge pull request #91 from EleenYang/master

update sample data

Commit:
update systemtest to reflect selectbook

Commit:
EOF

Commit:
style fix

Commit:
style fix

0blivious committed:
Merge pull request #92 from 0blivious/master

update systemtest to reflect selectbook

0blivious committed:
Update README.adoc

0blivious committed:
Update README.adoc

0blivious committed:
Update README.adoc

0blivious committed:
Update README.adoc

r4venz committed:
Merge pull request #13 from cs2103-ay1819s2-w11-4/master

190330

r4venz committed:
Model Tests:
1. ReviewTest
2. UniqueReviewListTest
3. ReviewBookNameContainsExactKeywordsPredicateTest

r4venz committed:
Parser Tests:
4. AddReviewCommandParserTest
5. DeleteReviewCommandParserTest
6. ListReviewCommandParserTest

r4venz committed:
SelectReviewCommand, parser, and relevant tests

r4venz committed:
Fixed config.json missing line feed at EOF problem

r4venz committed:
Command Tests
8. AddReviewCommandTest
9. DeleteReviewCommandTest
10.ListReviewCommandTest
11.SelectReviewCommandTest
12.ListAllReviewsCommandTest

joey-jiang6 committed:
Merge remote-tracking branch 'upstream/master'

r4venz committed:
UI Tests
13. BookBrowserPanelTest
14. ReviewListPanelTest
15. ReviewCardTest

EleenYang committed:
update data constraints

joey-jiang6 committed:
individual sort order

joey-jiang6 committed:
Update UniqueBookList.java

joey-jiang6 committed:
update sort function

EleenYang committed:
update the summary feature

EleenYang committed:
update tests for summary

joey-jiang6 committed:
update sort function

EleenYang committed:
update the user guide

joey-jiang6 committed:
fix style

joey-jiang6 committed:
Update UniqueBookList.java

EleenYang committed:
update userguide

EleenYang committed:
fix style

EleenYang committed:
fix style problems

EleenYang committed:
Update config.json

EleenYang committed:
fix coding style

0blivious committed:
Merge pull request #94 from EleenYang/master

update summary command and data constraints

r4venz committed:
style check

r4venz committed:
style check

r4venz committed:
style check

r4venz committed:
style check

0blivious committed:
Merge pull request #95 from Miaaaaa97/master

Update Tests

joey-jiang6 committed:
update sort

joey-jiang6 committed:
Merge remote-tracking branch 'upstream/master'

joey-jiang6 committed:
Update config.json

joey-jiang6 committed:
Update BookShelfParserTest.java

joey-jiang6 committed:
bug fix

joey-jiang6 committed:
Update BookShelfParserTest.java

joey-jiang6 committed:
Update BookShelfParserTest.java

joey-jiang6 committed:
Update SortBookCommand.java

Commit:
update UG

Commit:
style fix

Commit:
style fix

0blivious committed:
Merge pull request #96 from 0blivious/master

update UG

joey-jiang6 committed:
fix bugs

joey-jiang6 committed:
Merge remote-tracking branch 'upstream/master'

0blivious committed:
Merge pull request #93 from jiangqinhua/master

update sort

EleenYang committed:
update summary command

EleenYang committed:
update user doc

Commit:
update DG

r4venz committed:
Update UI image

r4venz committed:
Merge pull request #14 from cs2103-ay1819s2-w11-4/master

20190402

0blivious committed:
Merge pull request #98 from EleenYang/master

minor changes for summary command

Commit:
PPP

Commit:
Merge branch 'master' of https://github.com/cs2103-ay1819s2-w11-4/main

0blivious committed:
Merge pull request #97 from 0blivious/master

update DG

EleenYang committed:
update the developer guide (diagram)

Commit:
update PPP

Commit:
Merge branch 'master' of https://github.com/cs2103-ay1819s2-w11-4/main

EleenYang committed:
Merge branch 'master' into master

Commit:
fix

Commit:
test

Commit:
test

Commit:
upload UG and PPP

0blivious committed:
Merge pull request #99 from EleenYang/master

update the developer guide (diagram)

r4venz committed:
EditBook compatibility;
Update docs

r4venz committed:
Merge branch 'master' into master

0blivious committed:
Merge pull request #100 from 0blivious/master

update PPP, UG, DG, header html file

joey-jiang6 committed:
update DG and UG

update DG and UG, and fix some erros in sortbook command

joey-jiang6 committed:
Update DeveloperGuide.adoc

0blivious committed:
Merge pull request #101 from Miaaaaa97/master

Update UI image, docs, EditBook compatibility

joey-jiang6 committed:
Update UserGuide.adoc

joey-jiang6 committed:
update dg and ug

0blivious committed:
Update UserGuide.adoc

0blivious committed:
Merge pull request #102 from jiangqinhua/master

update DG and UG

r4venz committed:
Update window size and docs

r4venz committed:
Merge pull request #15 from cs2103-ay1819s2-w11-4/master

PPP

r4venz committed:
Merge branch 'master' of https://github.com/Miaaaaa97/main

0blivious committed:
Update AboutUs.adoc

0blivious committed:
Merge pull request #104 from cs2103-ay1819s2-w11-4/0blivious-patch-1

Update AboutUs.adoc

0blivious committed:
Merge pull request #103 from Miaaaaa97/master

Update window size and docs

Commit:
update UG PPP

Commit:
Merge branch 'master' of https://github.com/cs2103-ay1819s2-w11-4/main

0blivious committed:
Merge pull request #106 from 0blivious/master

update UG and PPP

EleenYang committed:
Update DG

EleenYang committed:
update dg

EleenYang committed:
Merge https://github.com/cs2103-ay1819s2-w11-4/main

EleenYang committed:
update about us & developer guide

EleenYang committed:
update developer guide

EleenYang committed:
update ppp

EleenYang committed:
update ppp

EleenYang committed:
update ppp

joey-jiang6 committed:
Create jiangqinhua.adoc

0blivious committed:
Merge pull request #107 from EleenYang/master

update developer guide

joey-jiang6 committed:
Update jiangqinhua.adoc

joey-jiang6 committed:
Update jiangqinhua.adoc

0blivious committed:
Merge pull request #108 from jiangqinhua/master

Create jiangqinhua.adoc

joey-jiang6 committed:
update system test

Commit:
update UI and UG

Commit:
fix

Commit:
fix

Commit:
update UG

0blivious committed:
Merge pull request #109 from jiangqinhua/master

update system test

Commit:
fix EOF

Commit:
update test

Commit:
Merge branch 'master' of https://github.com/cs2103-ay1819s2-w11-4/main

Commit:
fix test

0blivious committed:
Merge pull request #110 from 0blivious/master

update UI and UG

Commit:
bug dix, allow space in tag, allow 01 in rating

Commit:
update UG

Commit:
fix ug

0blivious committed:
Merge pull request #121 from 0blivious/master

fix bugs

joey-jiang6 committed:
bug fix

joey-jiang6 committed:
Update UserGuide.adoc

0blivious committed:
Merge pull request #130 from jiangqinhua/master

bug fix

Commit:
limit user input to 50 characters, limit each prefix to be supplied only once

Commit:
style fix

0blivious committed:
Merge pull request #131 from 0blivious/master

limit user input to 50 characters, limit each prefix to be supplied o…

EleenYang committed:
update addbook command parser

EleenYang committed:
Merge https://github.com/cs2103-ay1819s2-w11-4/main

r4venz committed:
Merge pull request #16 from cs2103-ay1819s2-w11-4/master

20190409

Commit:
update UI

r4venz committed:
debug

0blivious committed:
Merge pull request #132 from 0blivious/master

update UI

Commit:
updat ui

r4venz committed:
Update docs & main window min size problem

0blivious committed:
Merge pull request #134 from Miaaaaa97/master

debug user guide and main window problem

0blivious committed:
Merge branch 'master' into master

0blivious committed:
Merge pull request #133 from 0blivious/master

updat ui

EleenYang committed:
change the data constraints

book name and review title now only allows alphanumeric, space, '*', ',', '.', '?', ''' and '&'.

0blivious committed:
Merge pull request #135 from EleenYang/master

change book name and review title constraints

Commit:
update UI

0blivious committed:
Merge pull request #136 from 0blivious/master

update UI

r4venz committed:
Merge pull request #17 from cs2103-ay1819s2-w11-4/master

190410

Commit:
no message

r4venz committed:
adjust UI

0blivious committed:
Merge pull request #137 from 0blivious/master

minor fix

r4venz committed:
travis

0blivious committed:
Merge pull request #138 from Miaaaaa97/master

Adjust UI and tag color

r4venz committed:
Merge pull request #18 from cs2103-ay1819s2-w11-4/master

merge

joey-jiang6 committed:
update UG and PPP

Commit:
update UG

0blivious committed:
Merge pull request #140 from jiangqinhua/master

update UG and PPP

Commit:
Merge branch 'master' of https://github.com/cs2103-ay1819s2-w11-4/main

0blivious committed:
Merge pull request #141 from 0blivious/master

update UG

Commit:
update UG

0blivious committed:
Merge pull request #142 from 0blivious/master

update UG

joey-jiang6 committed:
Update DeveloperGuide.adoc

Commit:
update DG

0blivious committed:
Merge pull request #144 from 0blivious/master

update DG

joey-jiang6 committed:
Update DeveloperGuide.adoc

0blivious committed:
Merge branch 'master' into master

0blivious committed:
Merge pull request #143 from jiangqinhua/master

Update DeveloperGuide.adoc

r4venz committed:
Update DG

EleenYang committed:
update data for tests

0blivious committed:
Merge pull request #145 from Miaaaaa97/master

Update DG Diagrams

joey-jiang6 committed:
Update DeveloperGuide.adoc

0blivious committed:
Merge pull request #146 from jiangqinhua/master

Update DeveloperGuide.adoc

r4venz committed:
Update DG

joey-jiang6 committed:
update documents

0blivious committed:
Merge pull request #147 from jiangqinhua/master

update documents

EleenYang committed:
refractor

EleenYang committed:
Merge branch 'master' into master

0blivious committed:
Merge pull request #148 from EleenYang/master

update storage part and test data

EleenYang committed:
code fix

r4venz committed:
update ui

joey-jiang6 committed:
Update jiangqinhua.adoc

r4venz committed:
Merge branch 'master' into pr/19

joey-jiang6 committed:
Merge remote-tracking branch 'upstream/master'

r4venz committed:
Update UI pic

joey-jiang6 committed:
Update AddReviewCommandParser.java

0blivious committed:
Merge pull request #150 from jiangqinhua/master

Update jiangqinhua.adoc

0blivious committed:
Merge pull request #151 from Miaaaaa97/master

update ui pic

r4venz committed:
modify reviewcard

0blivious committed:
Merge pull request #152 from Miaaaaa97/master

modify reviewcard

Commit:
update PPP

0blivious committed:
Merge pull request #153 from EleenYang/master

style fix

0blivious committed:
Merge pull request #154 from 0blivious/master

update PPP

EleenYang committed:
Change the UI picture

EleenYang committed:
Merge https://github.com/cs2103-ay1819s2-w11-4/main

EleenYang committed:
change a file name

0blivious committed:
Merge pull request #155 from EleenYang/master

Change the UI picture

r4venz committed:
Update miaaaaa97.adoc

r4venz committed:
Merge pull request #20 from cs2103-ay1819s2-w11-4/master

190415

r4venz committed:
Update DG

r4venz committed:
Merge branch 'master' of https://github.com/Miaaaaa97/main

EleenYang committed:
fix bug in list book

r4venz committed:
bug fix

r4venz committed:
Merge pull request #156 from Miaaaaa97/master

Update DG, PPP and minor fix of bugs

r4venz committed:
add diagram in PPP

0blivious committed:
Merge pull request #157 from Miaaaaa97/master

add diagram in PPP

r4venz committed:
Update DeveloperGuide.adoc

EleenYang committed:
update UG DG

EleenYang committed:
Create listBookSqDiagram.png

EleenYang committed:
update image

Commit:
update DG

EleenYang committed:
Merge pull request #158 from EleenYang/master

update my ppp & fix a bug in listbook command

Commit:
update DG and diagram

Commit:
Merge branch 'master' of https://github.com/cs2103-ay1819s2-w11-4/main

# Conflicts:
#	docs/DeveloperGuide.adoc

EleenYang committed:
Update eleenyang.adoc

Commit:
update image

Commit:
bug ifx

EleenYang committed:
Merge pull request #159 from EleenYang/master

Update eleenyang.adoc

Commit:
fix

Commit:
Merge branch 'master' of https://github.com/cs2103-ay1819s2-w11-4/main

Commit:
change image format

Commit:
crop image

Commit:
crop image

Commit:
updage DG

Commit:
style fix

0blivious committed:
Merge pull request #160 from 0blivious/master

update DG

joey-jiang6 committed:
update docs

joey-jiang6 committed:
Merge pull request #161 from jiangqinhua/master

update docs

joey-jiang6 committed:
Update DeveloperGuide.adoc

joey-jiang6 committed:
Update DeveloperGuide.adoc

joey-jiang6 committed:
Merge pull request #162 from jiangqinhua/master

update docs

joey-jiang6 committed:
Update jiangqinhua.adoc

joey-jiang6 committed:
Merge pull request #163 from jiangqinhua/master

Update jiangqinhua.adoc

joey-jiang6 committed:
Revert "Update jiangqinhua.adoc"

Commit:
update UG

Commit:
Merge branch 'master' of https://github.com/cs2103-ay1819s2-w11-4/main

joey-jiang6 committed:
add demo pic

joey-jiang6 committed:
Merge pull request #165 from jiangqinhua/master

add demo pic

joey-jiang6 committed:
Merge pull request #164 from cs2103-ay1819s2-w11-4/revert-163-master

Revert "Update jiangqinhua.adoc"

EleenYang committed:
update diagrams

EleenYang committed:
Merge pull request #166 from EleenYang/master

update diagrams

Commit:
update UG and minor bug fix

Commit:
fix

0blivious committed:
Merge pull request #167 from 0blivious/master

update UG

Commit:
fix

Commit:
Merge branch 'master' of https://github.com/cs2103-ay1819s2-w11-4/main

Commit:
fix

r4venz committed:
Merge pull request #21 from cs2103-ay1819s2-w11-4/master

merge

r4venz committed:
Update DG

Commit:
change tag

r4venz committed:
Update UG

r4venz committed:
Merge pull request #168 from Miaaaaa97/master

Update DG Diagram
